### PR TITLE
Update start-working-in-bcgov-github-organization.md

### PR DESCRIPTION
**Pull request by Shivanganii**
- 1st publishable draft 
- Grade 7 readability 
- Prior feedback added prior to PR

### CODE REVIEW

File: docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md [hunk 1]
```diff
@@ -62,6 +62,15 @@ In both cases, the basic steps to release the code are similar, while the implic
 
 If you are intending to maintain an active project, make sure to establish the appropriate processes and terms to manage contributions.
 
+## Integrating GitHub Apps
+
+We have large volumes of asks about adding GitHub Apps to our organization’s repository. To manage the asks, we created a request process through the devops-requests repository.
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md:67]: please link to the devops request repository here.

File: docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md [hunk 2]
```diff
@@ -62,6 +62,15 @@ In both cases, the basic steps to release the code are similar, while the implic
...
+If the existing applications, like SonarCloud, does not work for you or your team, please create an issue. We need:
```
Comment (jenreiher-bcgov) [inline docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md:68]: we've recently decided not to keep using sonar cloud. @MonicaG is there a better example of one we get requests for commonly?

**Review by jenreiher-bcgov — CHANGES_REQUESTED** [review]
Looking great! just a couple of small tweaks :)

File: docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md (hunk 2 above)
Comment (MonicaG) [reply docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md:68]: Maybe just take the example out?

Comment (MonicaG) [reply docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md:68]:
```suggestion
If the existing applications don't work for you or your team, please create an issue. We need: 
```

File: docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md (hunk 1 above)
Comment (MonicaG) [reply docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md:67]: Here is the link to the specific DevOps request for requesting a GitHub App. I've reworked the sentence to use the link. Please feel free to revert back to previous sentence.


```suggestion
We have large volumes of asks about adding GitHub Apps to our organization’s repository. Request a GitHub App to be installed from the BCDevOps Request Issue: [Request for integrating a GitHub App](https://github.com/BCDevOps/devops-requests/issues/new?assignees=MonicaG%2C+oomIRL%2C+SHIHO-I&labels=github-app%2C+pending&projects=&template=github_integration_request.md&title=).
```

**Review by MonicaG — CHANGES_REQUESTED** [review]

**Pull request merged**